staging_service/import_specifications/individual_parsers.py

```diff
@@ -351,15 +357,26 @@ def parse_dts_manifest(path: Path, validator: Draft202012Validator) -> ParseResu
     results = {}
     try:
         with open(path, "r") as manifest:
-            manifest_json = json.load(manifest)
-        if not isinstance(manifest_json, dict):
+            manifest = json.load(manifest)
+        if not isinstance(manifest, dict):
             return _error(Error(ErrorType.PARSE_FAIL, "Manifest is not a dictionary", spcsrc))
-        for err in validator.iter_errors(manifest_json):
+        for err in validator.iter_errors(manifest):
             err_str = err.message
             err_path = list(err.absolute_path)
             if err_path:
                 err_str += f" at {err_path}"
             errors.append(Error(ErrorType.PARSE_FAIL, err_str, spcsrc))
+        if not errors:
+            results = defaultdict(list)
+            instructions = manifest[_DTS_INSTRUCTIONS_KEY]
+            for resource_obj in instructions[_DTS_INSTRUCTIONS_OBJECTS_KEY]:
+                datatype = resource_obj[_DTS_INSTRUCTIONS_DATATYPE_KEY]
+                parameters = frozendict(resource_obj[_DTS_INSTRUCTIONS_PARAMETERS_KEY])
```

Comment:
So the DTS has do know the app parameter names for each datatype it processes? IOW the DTS is coupled to the apps?

Reply:
I also assume the file names / locations in the staging area are in the parameters? They can't be anywhere else...

Reply:
Yes to both. We talked about doing some references from the parameters to file paths, but if we do that'll be in a later version.

Reply:
Well, that seems less than ideal, but if you've agreed that that's how it'll work so be it